docs/core/diagnostics/built-in-metrics-aspnetcore.md

```diff
@@ -1,41 +1,81 @@
+  - [Instrument: `aspnetcore.rate_limiting.queued_requests`](#instrument-aspnetcorerate_limitingqueued_requests)
+  - [Instrument: `aspnetcore.rate_limiting.request.time_in_queue`](#instrument-aspnetcorerate_limitingrequesttime_in_queue)
+  - [Instrument: `aspnetcore.rate_limiting.requests`](#instrument-aspnetcorerate_limitingrequests)
 - [Meter: `Microsoft.AspNetCore.Server.Kestrel`](#meter-microsoftaspnetcoreserverkestrel)
-  * [Instrument: `kestrel.active_connections`](#instrument-kestrelactive_connections)
-  * [Instrument: `kestrel.connection.duration`](#instrument-kestrelconnectionduration)
-  * [Instrument: `kestrel.rejected_connections`](#instrument-kestrelrejected_connections)
-  * [Instrument: `kestrel.queued_connections`](#instrument-kestrelqueued_connections)
-  * [Instrument: `kestrel.queued_requests`](#instrument-kestrelqueued_requests)
-  * [Instrument: `kestrel.upgraded_connections`](#instrument-kestrelupgraded_connections)
-  * [Instrument: `kestrel.tls_handshake.duration`](#instrument-kestreltls_handshakeduration)
-  * [Instrument: `kestrel.active_tls_handshakes`](#instrument-kestrelactive_tls_handshakes)
+  - [Instrument: `kestrel.active_connections`](#instrument-kestrelactive_connections)
+  - [Instrument: `kestrel.connection.duration`](#instrument-kestrelconnectionduration)
+  - [Instrument: `kestrel.rejected_connections`](#instrument-kestrelrejected_connections)
+  - [Instrument: `kestrel.queued_connections`](#instrument-kestrelqueued_connections)
+  - [Instrument: `kestrel.queued_requests`](#instrument-kestrelqueued_requests)
+  - [Instrument: `kestrel.upgraded_connections`](#instrument-kestrelupgraded_connections)
+  - [Instrument: `kestrel.tls_handshake.duration`](#instrument-kestreltls_handshakeduration)
+  - [Instrument: `kestrel.active_tls_handshakes`](#instrument-kestrelactive_tls_handshakes)
 - [Meter: `Microsoft.AspNetCore.Http.Connections`](#meter-microsoftaspnetcorehttpconnections)
-  * [Instrument: `signalr.server.connection.duration`](#instrument-signalrserverconnectionduration)
-  * [Instrument: `signalr.server.active_connections`](#instrument-signalrserveractive_connections)
+  - [Instrument: `signalr.server.connection.duration`](#instrument-signalrserverconnectionduration)
+  - [Instrument: `signalr.server.active_connections`](#instrument-signalrserveractive_connections)
+
+## Meter: `Microsoft.AspNetCore.HeaderParsing`
+
+### Instrument: `aspnetcore.header_parsing.parse_errors`
+
+| Name | Instrument Type | Unit (UCUM) | Description |
+|--|--|--|--|
+| `aspnetcore.header_parsing.parse_errors` | Counter | `{parse_error}` | Number of errors that occurred when parsing HTTP request headers. |
+
+| Attribute | Type | Description | Examples | Presence |
+|--|--|--|--|--|
+| `aspnetcore.header_parsing.header.name` | string | The header name. | `Content-Type` | Always |
+| `error.type` | string | The error message. | `Unable to parse media type value.` | Always |
+
+Available starting in: .NET 8.0.
```

Comment:
I'm note sure whether this is correct or not. I think it would be better to specify here a version of dotnet/extensions the metric is available in. @IEvangelist Do you know if we will have special name to reference dotnet/extensions, like .NET Extensions 8.0?

Reply:
Everything in `dotnet/extensions` falls under the `.NET Platform Extensions` moniker in docs, in this case we can simply say `.NET 8`.